<?xml version="1.0"?>
<launch>
  <node pkg="BP" type="positioner" name="positioner" output="screen">
    <remap from="depth_rect" to="/camera/depth/image_raw" />
    <remap from="cam_info" to="/camera/depth/camera_info" />
  </node>
</launch>
